db/src/Pos/DB/Block/Slog/Logic.hs

```diff
@@ -238,7 +240,8 @@ slogApplyBlocks k (ShouldCallBListener callBListener) blunds = do
     -- If the program is interrupted at this point (after putting blunds
     -- in BlockDB), we will have garbage blunds in BlockDB, but it's not a
     -- problem.
-    bListenerBatch <- if callBListener then onApplyBlocks fixedNM blunds
+    let nm = makeNetworkMagic pm
```

Comment:
If the only thing we do with the `protocolMagic` value is create a `networkMagic` from it, why not accept it directly?

Reply:
That's true. My mistake.